### PR TITLE
Replace missed instances of "log" with "sign"

### PR DESCRIPTION
**Pull request by Joehuu**
~Don't know how to link tournaments to sign in, so I edited without it. I might miss other instances.~

### CODE REVIEW

**Comment by peppy** [comment]
Probably want to add invalidation comments on other localisations

**Comment by TPGPL** [comment]
Polish is good

**Comment by kj415j45** [comment]
⚠ Request Change: Remove notes in zh locale files since nothing changed.

**Comment by kj415j45** [comment]
Besides, there's a change in `en/Tournaments.php`: `:login` -> `sign in`. I don't think this is correct.

(I don't create Line comment because my Chrome on mobile crash when loading so much changes :v)

File: resources/lang/en/tournament.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
             'cancel' => 'Cancel Registration',
         ],
         'entered' => 'Your are registered for this tournament.<br><br>Note that this does not mean you have been assigned to a team.<br><br>Further instructions will be sent to you via email closer to the tournament date, so please ensure your osu! account\'s email address is valid!',
-        'login_to_register' => 'Please :login to view registration details!',
+        'login_to_register' => 'Please sign in to view registration details!',
```
Comment (nanaya) [inline resources/lang/en/tournament.php]: no need to change this one. The text uses whatever `users.anonymous.login_link` is.

Don't forget to remove comment for other locales.

**Pull request merged**